### DISCUSSION TITLE
ReferenceError: process is not defined

### DISCUSSION THREAD

**Discussion by wlwatkins** [wailsapp/wails#1874] [Q&A]
Hi, 

I'm trying to use [zeromq ](https://github.com/zeromq/zeromq.js/) in the front end to receive data directly in javascript (using svelte) from another script

but I'm getting `ReferenceError: process is not defined` when try the example

From my serarch, it is because process is only on the serevr side and is not available client side. I found some hints to send the process to the client side, but so far nothing has worked. 

any clues?

**Comment by leaanthony** [comment]
Isn't process a node thing? I'd highly recommend doing this in Go.

**Reply by wlwatkins** [reply]
It is apparently. I intend to use it to stream video (numpy array) from an outside python script. My idea was to open a tcp channel between the front end and the python script, completely bypassing the go back end. I'd rather use go to be honest (i pretty much hate typescript/javascript). How would recommend streaming data from go back end to the front end? A few fps would be enough, but through the zmq I reach the limit of 30 fps from the webcam (converted with opencv)

**Reply by leaanthony** [reply]
That's a good question and not one I think I can answer immediately. @stffabi may have more of an idea around which versions of webview support streaming. If you're ok with polling from the frontend, you could use a custom AssetHandler to ask Go for the current frame as an image and display it.

What platform is this for?

**Reply by wlwatkins** [reply]
that could work,  I'll lookat your doc for details on this. I'm on windows and for windows. just for precision’s sake, I'm not streaming video as one would stream an mp4 to the frontend. Rather, I need to quickly refresh a heatmap from an array. thanks

**Reply by stffabi** [reply]
Response streaming big files through AssetsHandler (e.g. a big video) is currently only supported on Linux. I hope we could add support for Windows and MacOS for v2.1
